tools/findcheckpoint/go.mod

```diff
@@ -0,0 +1,10 @@
+module findcheckpoint
+
+go 1.12
+
+require (
+	github.com/Qitmeer/qitmeer v0.0.0-20190928033548-1aed0cf3e311
```

Comment:
the version is suspicious that not the latest, explicit version might be better.